packages/shared.test/src/stubs/actions/moveOffice.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { FauxDataStoreBatch } from "../../FauxFoundry/FauxDataStoreBatch.js";
+import type { JustProps } from "../../FauxFoundry/typeHelpers/JustProps.js";
+import type { TH_ApplyActionRequestV2 } from "../../FauxFoundry/typeHelpers/TH_ApplyActionRequestV2.js";
+import type { MoveOffice } from "../actionsTypes.js";
+import { officeObjectType } from "../objectTypeV2.js";
+
+export function moveOfficeImpl(
```

Comment:
This is the magic. Running this action in the OSDK actually updates the in memory store.